src/CMakeLists.txt

```diff
@@ -215,9 +215,11 @@ register_migraphx_ops(
     rsqrt
     run_on_target
     scalar
-    scatter_add
-    scatter_mul
-    scatter_none
+    scatter_elements_none
+    scatter_elements_add
```

Comment:
I thought I'd go along with the ONNX spec since [Scatter](https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/blob/a09dc50215f91c5ec2c0d6b2afd9fa0ed45f931c/src/targets/gpu/jit/scatternd.cpp#L54) has been deprecated in favor of ScatterElements.
If you find the change really grating, I'll revert the names, just confirm it one more time please.

Reply:
Yea revert the names back to scatter.